src/views/assignment-edit.mustache

```diff
@@ -0,0 +1,46 @@
+{{> navbar}}
+
+<body>
+
+    <br>
+    <div class="alert alert-danger w-75 {{^show_error}}d-none{{/show_error}}" style="float: none; margin: 0 auto;">
+        <strong>Error!</strong> {{error}}</a>
+    </div>
+    {{#show_error}}<br>{{/show_error}}
+
+    <h1 align="center">Edit Auto-Assignment "{{old_name}}"!</h1>
+    <br>
+    <div class="w-75" style="float: none; margin: 0 auto;">
+        <form action="" method="post">
+            <div class="form-group">
+                Select Target Device
+                <select class="form-control" name="device" required>
+                    <option value="" {{#nothing_selected}}selected{{/nothing_selected}} disabled hidden>Choose a Device</option>
+                    {{#devices}}
+                        <option value="{{uuid}}" {{#selected}}selected{{/selected}}>{{uuid}}</option>
+                    {{/devices}}
+                </select>
+            </div>
+            <div class="form-group">
+                Select Destination Instance
+                <select class="form-control" name="instance" required>
+                    <option value="" {{#nothing_selected}}selected{{/nothing_selected}} disabled hidden>Choose a Instance</option>
+                    {{#instances}}
+                        <option value="{{name}}" {{#selected}}selected{{/selected}}>{{name}}</option>
+                    {{/instances}}
+                </select>
+            </div>
+            <div class="form-group">
+                Time (Empty for "On Complete")
+                <input type="text" class="form-control" name="time" value="{{time}}" minlength="7" maxlength="8" placeholder="HH:MM:SS">
```

Comment:
Conversion of time (int) to string is not working. It displays the int value here instead of a time string

Reply:
In DataTable it's now correct, but on edit page it still shows the int value
<img width="388" alt="Screenshot 2020-09-08 at 09 50 02" src="https://user-images.githubusercontent.com/194908/92448542-0a2a2780-f1b9-11ea-9521-9b8f631919a2.png">

Reply:
Should be fixed in the next push